magento/adobe-stock-integration
# Media gallery synchronization should consider file modification time &&  Performed media gallery && media content synchronization in batches

### Description (*)


### Fixed Issues (if relevant)

1. magento/adobe-stock-integration#1339: Media gallery synchronization should consider file modification time
2. magento/adobe-stock-integration#1337:
Media gallery synchronization should be performed in batches
3.  magento/adobe-stock-integration#1338: Media content synchronization should be performed in batches
4. magento/adobe-stock-integration#1340: Media content synchronization should consided entity updated at value

### Manual testing scenarios (*)
1. Run `bin/magento media-gallery:sync`
2. `php bin/magento media-content:sync`

Media gallery synchronization executed with batch of files 
Verify all files synchronized correctly

Media gallery synchronization should be performed only for files that were modified since the last synchronization execution
